02 - Web UI Template/CognitiveSearch.UI/Search/DocumentSearchClient.cs

```diff
@@ -1,24 +1,29 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using Azure;
+using Azure.Search.Documents;
+using Azure.Search.Documents.Indexes;
+using Azure.Search.Documents.Indexes.Models;
+using Azure.Search.Documents.Models;
+using Azure.Storage;
+using Azure.Storage.Blobs;
+using Azure.Storage.Sas;
 using Microsoft.ApplicationInsights;
-using Microsoft.Azure.Search;
-using Microsoft.Azure.Search.Models;
 using Microsoft.Extensions.Configuration;
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Net;
 using System.Threading.Tasks;
-using Microsoft.WindowsAzure.Storage.Blob;
-using Microsoft.WindowsAzure.Storage.Auth;
 
 namespace CognitiveSearch.UI
 {
     public class DocumentSearchClient
     {
         private IConfiguration _configuration { get; set; }
-        private SearchServiceClient _searchClient;
-        private ISearchIndexClient _indexClient;
+        private SearchIndexClient _searchIndexClient;
+        private SearchClient _searchClient;
         private string searchServiceName { get; set; }
         private string apiKey { get; set; }
```

Comment:
These can just be read only fields instead of private properties.

Reply:
`_isPathBase64Encoded` should specify an access modifier and be renamed if it is not meant to be private.

Reply:
FYI, I know a lot of these issues weren't touched as part of this PR, but I am just mentioning them here 😄 
-`_searchId` - does this need to be public? If so, we should make it a property and use PascalCase naming.
Same thing for `errorMessage`.